### DISCUSSION TITLE
How to animate rings?

### DISCUSSION THREAD

**Discussion by victormihalache** [tailwindlabs/tailwindcss#6010] [Help] [answered]
Given the following `div`:

```html
<div class="h-auto w-max m-4 p-2 rounded bg-red-400 hover:ring-4 hover:ring-red-200 hover:cursor-pointer">
  This is a hoverable div
</div>
```

How can the ring be animated so hovering the `div` looks smoother?

**Comment by NovusOpus** [comment, marked as answer]
You mean like [this(TailwindPlay)](https://play.tailwindcss.com/wvs0sqmfuz)?

Animated Code:

```HTML
<div class="h-auto w-max m-4 p-2 rounded bg-red-400 hover:ring-4 hover:ring-red-200 transition duration-700 hover:cursor-pointer">
  This is a hoverable div
</div>
```
Another example:
```HTML
<div class="h-auto w-max m-4 p-2 rounded bg-red-400 hover:ring-4 hover:ring-red-200 transition ease-out duration-700 hover:cursor-pointer">
  This is a hoverable div
</div>
```
You can play around using different values until you get the effect you want. Also if you want to learn more or check all the options available, take a look at the [TRANSITIONS](https://tailwindcss.com/docs/transition-property) AND [ANIMATION](https://tailwindcss.com/docs/animation)  in the Tailwind CSS Docs. They are well explained and with examples.

**Reply by victormihalache** [reply]
This is exactly what I was looking for, thank you for also guiding me to wards other resources 👍🏼
